mkt/purchase/webpay.py

```diff
@@ -206,6 +228,20 @@ def simulated_postback(contrib, trans_id):
     return http.HttpResponse(trans_id)
 
 
+def free_postback(request, contrib, trans_id, user_profile):
+    log.info('Got free product postback: fulfilling pruchase for contrib={c}; '
```

Comment:
these strings should be unicode literals like ``u'Got free...'`` because the stringified user_profile object might contain non-ascii chars (I think)